Tests/Functional/AbstractTestCase.php

```diff
@@ -26,164 +26,19 @@
 /**
  * Class AbstractTestCase.
  */
-abstract class AbstractTestCase extends WebTestCase
+abstract class AbstractTestCase extends ParentTestBase
```

Comment:
The right name for all duplicated classes name extensions aliases is `Base`.

p.s. there are many more other cases with different alias naming